src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright 2020 Esri
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+
+namespace ArcGISEarth.AutoAPI.Utils
+{
+    public class AutomationAPIHelper
+    {
+        private const string CameraControllerName = "camera";
+        private const string FlightControllerName = "flight";
+        private const string LayerControllerName = "layer";
+        private const string LayersControllerName = "layers";
+        private const string WorkspaceControllerName = "workspace";
+        private const string SnapshotControllerName = "snapshot";
+        private const string DEFAULT_BASEURL = "http://localhost:8000";
+        private const string END_POINT = "/arcgisearth";
+
+        public string APIBaseUrl 
+        {
+            get { return GetBaseUrl(); }
+        }
+
+        #region Automation API Function Tasks
+
+        /// <summary>
+        /// Automation API Functions
+        /// </summary>
+        /// <returns>Automation API response message</returns>
```

Comment:
Remove this description or all methods add description.

Reply:
We should add comments for all the methods in this file. @gis2all @xuew8412